applications/Cook Protocol.md

```diff
@@ -0,0 +1,151 @@
+**Cage Chen**  
+Top 1% graduate from Carnegie Mellon University and a Silicon Valley engineering veteran. Lead engineer at several top Silicon Valley companies, such as DropBox and C3.AI. Currently serves as tech lead on frontend & backend development. Expert in Java, Python, Typescript development.  
+**KP Peng**  
+Stanford Engineering School Graduate. Technical product manager from United Technologies Corporation. Fluent in 6 languages. Has successfully delivered multiple large-scale projects using languages and technologies such as C#.Net, Java, SQL Server Integration Service, Microsoft Azure, Python. 4 years of experience in the blockchain space.  
+**Jikun Liao**  
+Senior software engineer in IoTeX. 8 years of experience in frontend and 2 years of experience in blockchain. Highly skilled in React/Redux/Saga/Hook, HTML/CSS/JavaScript/Typescript, React Native, Node/Express/Mongo, Vue.js/Vuex/Vuetify, Angular/ngrx, Solidity/Truffle/HardHat/Web3 JS/EtherJS.  
+**Past blockchain experiences**
+IoTex explorer  
+https://github.com/liaojikunwork/iotex-explorer  
+Cook Leverage uses Uniswap Flashswap and Compound to reach the effect of margin trading to magnify returns:  
+https://github.com/CookFinance/cook-leverage  
+Price-based and time-based token distribution and liquidity mining:  
+https://github.com/CookFinance/cook-distribution-and-reward  
+
+### Team Code Repos
+* https://github.com/CookFinance
+
+### Team LinkedIn Profiles
+* https://www.linkedin.com/in/adrian-p-6564771b5/
+* https://www.linkedin.com/in/cage-chen-17a557107/
+* https://www.linkedin.com/in/jikun-liao-794b61162/
+* https://www.linkedin.com/in/kunpeng89/
+
+## Development Roadmap :nut_and_bolt: 
+
+### Overview
+* **Total Estimated Duration:** 2 months
+* **Full-time equivalent (FTE):**  2 FTE
+* **Total Costs:** 14k DAI
+
+### Milestone 1 Initialize Parachain & Frontend Implementation
+* **Estimated Duration:** 2 weeks
+* **FTE:**  2
+* **Costs:** 4k DAI
+
+| Number | Deliverable | Specification |
+| ------------- | ------------- | ------------- |
+| 0a. | License | Apache 2.0 / MIT / Unlicense |
+| 0b. | Documentation | We will provide both inline documentation of the code and a basic tutorial that explains how a user can (for example) spin up one of our Substrate nodes. Once the node is up, it will be possible to send test transactions that will show how the new functionality works. |
+| 0c. | Testing Guide | The code will have unit-test coverage (min. 70%) to ensure functionality and robustness. In the guide we will describe how to run these tests | 
+| 0d. | Article/Tutorial | We will write an article or tutorial that explains the work done as part of the grant. 
+| 1. | Frontend | We will implement frontend using React.js so that:  1. Users can connect to the polkadot wallet.  2. The fund manager can initialize a fund, and allocate assets via Polkaswap.  3. The investor can browse the fund list, check the performance details of a fund, and invest in a fund. |  
```

Comment:
Is polkaswap actually fully open source? It would be nice if you could link to the code running on the blockchain that you are planning to use here.

Reply:
We double checked the codebase of polkaswap and realized that it is far from ready. So we decided to change the plan to use https://centrifuge.io, which has the lending functionality that could be integrated into our asset management solution. It is an open source project and its code can be found: https://github.com/centrifuge/centrifugechain